OAuth: add docs for disableAutoLogin param (#28398)
grafana/grafana: docs/sources/auth/overview.md

oauth_auto_login = true
```

> If you want to skip automatic OAuth login and sign in with a username and password, then add the `disableAutoLogin` parameter to the login URL. This disables signing in with OAuth.

Author: Forget about version availability note. This feature will be available in 7.4.

Reviewer: AH, yes. Definitely want to add that.

> If you want to skip automatic OAuth login and sign in with username and password, add `disableAutoLogin` parameter to the login URL. Signing in with OAuth will be disabled in this case.

Reviewer: ```suggestion
If you want to skip automatic OAuth login and sign in with a username and password, then add the `disableAutoLogin` parameter to the login URL. This disables signing in with OAuth.
```

> Example: `grafana.example.com/login?disableAutoLogin` or `grafana.example.com/login?disableAutoLogin=true`.

Reviewer: ```suggestion
Example: `grafana.example.com/login?disableAutoLogin` or `grafana.example.com/login?disableAutoLogin=true`
```

Reviewer: Great examples! We need more examples throughout the doc. I think they really help clarify items like this.
